### PR TITLE
Add release notes for 2.8.3

### PR DESCRIPTION
**Pull request by cannischan**
Please review if the notes are suitable for public consumption

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -9,17 +9,44 @@ description: >
...
+### New
+
+Data Plane
+
+* Add support for bottlerocket
+* Check whether the block device directory (usually /var/lib/storageos/volumes) is mounted with `nodev` and/or `ro`, and if so attempt to remount the filesystem `rw` and with `dev` access
+
+k8s
+
+* Ability to disable the scheduler
```
Comment (Mojachieee) [inline docs/release-notes.md:34]:
```suggestion
* The scheduler extender, that attempts to place workloads on the same nodes as volumes, can now be disabled
```

File: docs/release-notes.md [hunk 2]
```diff
@@ -9,17 +9,44 @@ description: >
...
+
+* Increase the `LIO_DEVICE_TIMEOUT_SECS` to 300 seconds (5 minutes) and the `LIO_RETRY_LOOP_DURATION_SECS` to 240 seconds (4 minutes). This is more closely in line with iSCSI timeouts and affords us more time to retry IO on flaky/slow systems
+* Add environment variables so time-outs can be adjusted and tuned
+* Fix spelling mistake in alert log messages
+* Improve the clarity of the log messages which alert users that IO to the backend disk (fdatasync, preadv, pwritev and fallocate) is running unusually slowly
+* Fix an issue whereby creating a Ondat block device could erroneously fail because we fail to wait for the underlying kernel device to be available
+
+Control Plane
+
+* Reduce the amount of crash looping when a node goes down
```
Comment (Mojachieee) [inline docs/release-notes.md:48]:
```suggestion:
* The control plane will now crash loop less, when its pod is restarted.  
```

File: docs/release-notes.md [hunk 3]
```diff
@@ -9,17 +9,44 @@ description: >
...
+
```
Comment (Mojachieee) [inline docs/release-notes.md:49]: Looks like we're missing the k8s fixes: 
```
The following fixes apply to k8s clusters running v1.23, v1.24 and v1.25, the bugs did not apply to older cluster versions
* api-manager will now have permissions to use `podsecuritypolicies`
* api-manager now has the expected resource limits
* api-manager will no longer run as root 
* api-manager pods will now be spread across nodes
```

File: docs/release-notes.md (hunk 2 above)
Comment (cannischan) [reply docs/release-notes.md:48]: done

File: docs/release-notes.md (hunk 3 above)
Comment (cannischan) [reply docs/release-notes.md:49]: thanks!

**Review by Mojachieee — DISMISSED** [review]

File: docs/release-notes.md [hunk 4]
```diff
@@ -9,17 +9,52 @@ description: >
...
+k8s
+
+* The scheduler extender, that attempts to place workloads on the same nodes as volumes, can now be disabled
+
+### Fixed
+
+Data Plane
+
+* Increase the `LIO_DEVICE_TIMEOUT_SECS` to 300 seconds (5 minutes) and the `LIO_RETRY_LOOP_DURATION_SECS` to 240 seconds (4 minutes). This is more closely in line with iSCSI timeouts and affords us more time to retry IO on flaky/slow systems
```
Comment (chira001) [inline docs/release-notes.md:40]: Update to:

Increase the `LIO_DEVICE_TIMEOUT_SECS` to 300 seconds (5 minutes) and the `LIO_RETRY_LOOP_DURATION_SECS` to 240 seconds (4 minutes).  This provides additional flexibility for environments experiencing resource contention.

File: docs/release-notes.md [hunk 5]
```diff
@@ -9,17 +9,52 @@ description: >
...
 To upgrade from version 1.x to 2.x, contact Ondat [support](/docs/support) for assistance.
 
+## 2.8.3 - Release 2022-09-14
+
+### New
+
+Data Plane
+
+* Add support for bottlerocket
+* Check whether the block device directory (usually /var/lib/storageos/volumes) is mounted with `nodev` and/or `ro`, and if so attempt to remount the filesystem `rw` and with `dev` access
```
Comment (chira001) [inline docs/release-notes.md:30]: Change to:
Check whether the block device directory (usually /var/lib/storageos/volumes) supports creation of devices, and enable it if it is not already.

File: docs/release-notes.md (hunk 4 above)
Comment (cannischan) [reply docs/release-notes.md]: done

**Review by DavidMarchant — APPROVED** [review]

**Pull request merged**